packages/app/src/sandbox/eval/presets/svelte/index.js

```diff
@@ -23,6 +26,23 @@ export default function initialize() {
     { transpiler: babelTranspiler, options: babelOptions },
   ]);
 
+  sveltePreset.registerTranspiler(module => /\.css$/.test(module.path), [
+    { transpiler: styleProcessor },
+    {
+      transpiler: stylesTranspiler,
+      options: { hmrEnabled: false },
+    },
+  ]);
+
+  sveltePreset.registerTranspiler(module => /\.s[c|a]ss$/.test(module.path), [
+    { transpiler: sassTranspiler },
+    { transpiler: styleProcessor },
```

Comment:
Mini comment: this transpiler can be slow at times, if we don't necessarily need it we should maybe consider not using it, but it's just something to keep in mind.

Reply:
I mean I added it because we need it :/

I tried without it

Reply:
Ah, okay, good to know!